src/it/java/org/greencodeinitiative/creedengo/python/integration/tests/GCIRulesIT.java

```diff
@@ -273,4 +273,18 @@ void testGCI203_compliant() {
 
     }
 
+    @Test
+    void testGCI96() {
+        String filePath = "src/pandasRequireUsecols.py";
```

Comment:
integration test error when I launch "mvn verify" because this file doesn't exist. There is an error in the name : "src/pandasRequireUseCols.py" instaad of ""src/pandasRequireUsecols.py"
please launch "mvn verify" to check if tests controls are OK (this is the process launched by CI) and next you can check manually if you want with a "mvn verify -Pkeep-running"